sdk/src/main/java/io/dapr/client/domain/Job.java

```diff
@@ -0,0 +1,74 @@
+package io.dapr.client.domain;
+
+import com.google.protobuf.Message;
+
+/**
+ * A Job to schedule
+ *
+ * @param <T> The class type of Job data.
+ */
+public final class Job<T extends Message> {
+
+	private final String name;
+	
+	private String schedule;
+	
+	private Integer repeats;
+	
+	private String dueTime;
+	
+	private String ttl;
+	
+	private final T data;
+
+	/**
+	 * Constructor for Job
+	 * 
+	 * @param name name of the job to create
+	 */
+	public Job(String name, T data) {
```

Comment:
The constructor should include all required fields.

Reply:
@charan2628 ideally you should put all the data in constructor and have just "getters" without setters.

Reply:
@artur-ciocanu as per the spec only name and data are required, remaining are optional that's why I included only these two in constructor, can I two constructors one with mandatory fields and another with all fields?